config/schemas/config.yaml

```
@@ -8591,8 +8591,8 @@ properties:
           trivyOperator:
             $ref: '#/$defs/containerImage'
       nodeLocalDns:
-        title: node-local-dns stack image configuration
-        description: node-local-dns stack image configuration
+        title: Node-local-dns stack image configuration
+        description: Node-local-dns stack image configuration
```

NodeLocal DNSCache: https://kubernetes.io/docs/tasks/administer-cluster/nodelocaldns/

```suggestion
        title: NodeLocal DNSCache stack image configuration
        description: NodeLocal DNSCache stack image configuration
```